testing/scenario_app/ios/Scenarios/ScenariosUITests/GoldenImage.m

```diff
@@ -102,10 +102,11 @@ - (BOOL)compareGoldenToImage:(UIImage*)image {
 }
 
 NS_INLINE NSString* _platformName() {
+  NSString* systemVersion = UIDevice.currentDevice.systemVersion;
```

Comment:
This is the main change--`golden_bogus_font_text_iPhone 8_13.0_simulator.png` and `golden_bogus_font_text_iPhone 8_15.0_simulator.png` can both be checked in.